bip-path-templates.mediawiki

```diff
@@ -97,29 +97,29 @@ For Unit range, we say that range_start is equal to range_end, even though there
 
 Constraints:
 
-# For the avoidance of ambiguity, whitespace MUST NOT appear within the path template.
+# To avoid ambiguity, whitespace MUST NOT appear within the path template.
 # Commas within the Ranged index template MUST only appear in between index ranges.
-# For the avoidance of ambiguity, an index range that matches a single value MUST be specified as Unit range.
+# To avoid ambiguity, an index range that matches a single value MUST be specified as Unit range.
 # For Non-unit range, range_end MUST be larger than range_start.
-# If there are more than one index range within the Ranged index template, range_start of the second and any subsequent range MUST be larger than the range_end of the preceeding range.
-# For the avoidance of ambiguity, all representations of integer values larger than 0 MUST NOT start with character "0" (no leading zeroes allowed)
+# If there is more than one index range within the Ranged index template, range_start of the second and any subsequent range MUST be larger than the range_end of the preceeding range.
+# To avoid ambiguity, all representations of integer values larger than 0 MUST NOT start with character "0" (no leading zeroes allowed).
 # If hardened marker appears within any section in the path template, all preceding sections MUST also specify hardened matching.
-# For the avoidance of ambiguity, if hardened marker appears within any section in the path template, all preceding sections MUST also use the same hardened marker (either "h" or "'")
-# For the avoidance of ambiguity, trailing slashes (for example, "1/2/") and duplicate slashes (for example, "0//1") MUST NOT appear in the template
+# To avoid ambiguity, if a hardened marker appears within any section in the path template, all preceding sections MUST also use the same hardened marker (either "h" or "'").
+# To avoid ambiguity, trailing slashes (for example, "1/2/") and duplicate slashes (for example, "0//1") MUST NOT appear in the template.
 
-It may be desireable to have fully unambiguous encoding, where for each valid path template string, there are no other valid template string that matches the exact same set of paths. This would enable to compare templates for equality by simple string equality check, without any parsing.
+It may be desireable to have fully unambiguous encoding, where for each valid path template string, there is no other valid template string that matches the exact same set of paths. This would enable someone to compare templates for equality through a simple string equality check, without any parsing.
 
-To achieve full non-ambiguoussness, in addition to the rules above, one extra rule is needed:
+In addition to the rules above, one extra rule is needed:
```

Comment:
I feel that this phrase still needs a connection to the previous paragraph. Maybe, "To achieve this, ..." ?

Reply:
Agree. If the former is kept, then it would just need spelling correction to "ambiguousness" in all instances where that is used.